routines/cogs/role.py

```diff
@@ -147,22 +147,6 @@ async def rsmod(self, ctx):
 
     @rsmod.group()
     async def on(self, ctx, mod):
-        extras = {
-            'croid': discord.utils.get(self.bot.emojis, name='croid'),
-            'influence': discord.utils.get(self.bot.emojis, name='influence'),
-            'nosanc': discord.utils.get(self.bot.emojis, name='nosanc'),
-            'notele': discord.utils.get(self.bot.emojis, name='notele'),
-            'rse': discord.utils.get(self.bot.emojis, name='rse'),
-            'suppress': discord.utils.get(self.bot.emojis, name='suppress'),
-            'unity': discord.utils.get(self.bot.emojis, name='unity'),
-            'veng': discord.utils.get(self.bot.emojis, name='veng'),
-            'barrage': discord.utils.get(self.bot.emojis, name='barrage'),
-            'laser': discord.utils.get(self.bot.emojis, name='laser'),
-            'dart': discord.utils.get(self.bot.emojis, name='dart'),
-            'battery': discord.utils.get(self.bot.emojis, name='battery'),
-            'solo': discord.utils.get(self.bot.emojis, name='solo'),
-            'solo2': discord.utils.get(self.bot.emojis, name='solo2')
-        }
```

Comment:
This was unused, do we need it? If so, should we place it somewhere where all the methods can access it? Like some helper library.